core/controlplane/config/templates/cloud-config-worker

```diff
@@ -416,14 +428,15 @@ ssh_authorized_keys:
       content: |
         [Unit]
         Description=Pull and tag a mirror image for pause-amd64
-        After=docker.service
         Wants=docker.service
 
         [Service]
-        Restart=on-failure
+        Type=oneshot
```

Comment:
so that `docker.service` (and other dependent services) wait until `pause-amd64.service` finished. Good 👍